packages/cfpb-expandables/src/cfpb-expandables.less

```diff
@@ -209,3 +209,11 @@
     }
   } );
 }
+
+// Used when the set language reads right-to-left 
+[dir="rtl"] {
```

Comment:
Will this attribute appear on the top expandable element? Should we scope it to that? e.g.

```css
.o-expandable[dir="rtl"] {
```

(Or, perhaps better, move it up to between line 210-211 and make it `&[dir="rtl"] {`)

Reply:
Hmmm, I'm not sure if I'm missing something but I've tested both those options and neither worked. Through all of my trial and error, I've only had success in changing `o-expandable_header` which is what the button is attached to. I am open to different solutions if we feel they would be better, but so far this is the best I've come up with

Reply:
`o-expandable_header` would stay. I'm just asking what element `dir="rtl"` appears on?

The full selector would be:

```css
.o-expandable[dir="rtl"] {
    .o-expandable_header {
    …
    }
}
```

Though you should be able to compact that to 

```css
.o-expandable[dir="rtl"] .o-expandable_header { 
   … 
}
```

Reply:
Ah sorry, I misunderstood. `dir="rtl"` appears in the block `.block_-flush-top` which holds the expandable group within it. The problem is `dir="rtl"` is never applied to the `.o-expandable` element, which is why I put this code outside of it.

<img width="892" alt="image" src="https://user-images.githubusercontent.com/85513449/203425089-ff65e2c1-4af8-4f86-a7af-9a1cfb83bc55.png">

Reply:
Ah gotcha! Okay, looks like it's applied here to a `div` https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/_includes/templates/render_block.html#L14

So I'd probably make it:

```css
div[dir="rtl"] .o-expandable_header { 
  …
}
```

That way it doesn't have to check every element in the dom tree for the `dir` attribute, only the `div`s (which will still be a large number, but not everything).

---

Zooming out a bit, will the pages this appears on always have a language setting on the `<html>` element? 
This [W3C page](https://www.w3.org/International/questions/qa-html-dir) says:
> If the overall document direction is right-to-left, add dir="rtl" to the html tag.
>
> Below the html tag, only use the dir attribute on structural elements on the rare occasions when the base direction needs to change in order for the text to display correctly.

If this will be always used on pages where the whole page is `rtl`, than that attribute should be added to the `html` element and you could do:

```css
:root[dir="rtl"] .o-expandable_header { 
  …
}
```

@contolini since it looks like you wrote that accessible block, any thoughts on one or the other of the above?

Reply:
AFAIK we don't have a `dir` attribute on our root element because it reverses *all* content on the page and breaks the mega menu on smaller screens.

<img width="997" alt="Screen Shot 2022-11-23 at 10 54 44 AM" src="https://user-images.githubusercontent.com/1060248/203591431-e91d491c-444f-496d-9fd3-fa5b6da286d6.png">

Instead, we apply the attribute at the block level. This is not great and if we want to properly support RTL languages we should make our layouts direction-agnostic. Although before we do that, we should translate our header and footer content.

So with our current set-up, I think `div[dir="rtl"] .o-expandable_header {` is as specific as we can get to fix expandables.

Reply:
@brandon-rusk - I think what Ans suggests is a good change! 👍

Reply:
Awesome, thanks for the feedback, all! I have updated the PR.

@anselmbradford, is there any docs that need to be updated in the DS to show Expandables now handle right-to-left languages?

Reply:
Yeah wouldn't be a bad idea to add an example of this. You can do that by adding a section in https://github.com/cfpb/design-system/blob/main/docs/pages/expandables.md (follow the basic expandable format, but wrap it in `<div dir="rtl">…</div>`.